### PR TITLE
Fix/ms 614 content container width

### PR DESCRIPTION
**Pull request by lecosson**
Related to : https://oat-sa.atlassian.net/browse/MS-614
 
Fix content area header width style when navbar is hidden
  
#### How to test

- navigate to backoffice
- open page without navbar (for example, extensions list
- ensure content area header has a full width

#### Related PR's
none

### CODE REVIEW

**Review by anshul119 — APPROVED** [review]
Works as expected on NCCER on normal pages and with trees as well.

**Review by anshul119 — CHANGES_REQUESTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by anshul119 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: views/scss/inc/_section-container.scss [hunk 1]
```diff
@@ -120,6 +120,10 @@
     .content-container {
         border: none;
         @include simple-flex-box();
+        max-width: 100%;
```
Comment (andrey-ponamarev) [inline views/scss/inc/_section-container.scss:123]: Why do we need `max-width` if markup is using flex box?

Comment (lecosson) [reply views/scss/inc/_section-container.scss:123]: Andrii, it avoid a dirty behavior of some components when xtra-large content rendered

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**